plutus-playground/Changelog.md

```diff
@@ -0,0 +1,10 @@
+# plutus-playground Changelog
```

Comment:
I'm not sure if we want to do changelogs yet: we don't really have releases, and they're quite a lot of work.

Reply:
Yeah but we have a product that's been released - we should at least keep track of changes since the first release so that we have something to show when the next release happens. This is just for the user facing side of things.